lesson07/assignment/lesson7/src/components/RecipesView.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div id="app">
+    <main class="main__recipes">
+
+      <!-- recipe of the day card (same as index) -->
+      <div class="container">
+        <div class="row">
+          <div class="col-12 p-5">
+            <article class="card">
+              <a class="card__link" href="recipe.html">
+                <h3 class="card__title">Today's Recipe</h3>
+                <h4 class="card__subtitle">Seeded Rye Sourdough Boule</h4>
+                <img class="card__img" src="@/assets/bread-2.jpeg" alt="bread goes here">
+            </a>
+            </article>
+          </div>
+        </div>
+
+        <!-- a bottom row with more recipes -->
+        <div class="row">
+          <div class="col-lg-8 col-12 p-3">
+            <article class="recipes">
+              <h3 class="recipes__title">Base Recipes</h3>
+              <ul>
+                <li class="recipes__item" v-for="recipe in baseRecipes" v-bind:key="recipe.id">
```

Comment:
I don't think that your baseRecipes object has an id.  If you had some type of interactivity with the items in this loop, it may not work how you think it should.  For example if each recipe had a delete or hide button, there would be a bug that might cause it to target the wrong item in the list.

You can fix the loop by adding an iterating id and using that as the key.

`<li class="recipes__item" v-for="(recipe, id) in xRecipes" :key="id">
                  <a :href="recipe.link" class="recipes__link"> >> {{recipe.name}}</a>
                </li>`